tests/hardware_tests/test_gaze_env.py

```diff
@@ -55,27 +56,39 @@ def test_gaze():
 
     test_target_objects = ["cup", "plush_lion", "plush_bear"]
     test_spot = Spot("GazeEnvHardwareTest_StaticGaze")
-    test_result = None
+    test_results = []  # type: List[Dict]
```

Comment:
you could also do: `test_results: List[Dist] = []`

Reply:
For rest of the PR, I have used `var = val # type: <dtype>` format. I guess let's use the existing one for consistency of the PR?